xml/cap_admin_service_broker.xml

```diff
@@ -244,7 +244,7 @@ kube:
   <title>Deploying the MySQL Chart</title>
 
   <para>
-   &productname; includes charts for MySQL and PostgreSQL (see
+   &productname; includes charts for &mariadb; and &postgresql; (see
```

Comment:
Probably need to revert this if we're not replacing all the occurrences.

Reply:
I would lean towards reverting this change (and the change above) so the final output remains `MySQL` in this service broker chapter since:
- things like the chart itself uses mysql as its naming and would be better for consistency
- there are quite a few other instances that would need to be changed